themes/default/content/blog/nodejs-binaries-with-pkg/index.md

```diff
@@ -0,0 +1,131 @@
+---
+title: "Node.js Native Binary Compilation Using vercel/pkg"
+date: 2022-09-13
+meta_desc: "Using Vercel pkg to create native binaries for nodejs providers"
+
+#meta_image: infra-observability.png
+authors:
+- daniel-bradley
+tags:
+- engineering
+- nodejs
+- publishing
+- providers
+
+meta_image: "meta.png"
+---
+
+It's well known that you can write Pulumi programs in loads of languages, but what about the providers behind the scenes which handle communication with various cloud services?
```

Comment:
I'd suggest fleshing out the intro section more, and setting some context. This is an engineering deep dive that some readers might opt to skip, so it would be good to cover the high-level points in the first couple of paragraphs, and then transition into the technical details after that.

The questions in my mind right now are:
* What is a provider?
* What is a component provider?
* Why might you want to write a provider in Typescript?

We probably have existing content that covers most of this in detail, so a few links could go a long way here.

Reply:
I've made it a little less about providers, because it's not really about them specifically. Hopefully the reworked intro covers more of of the "Why Typescript" and "Why binary packages" part.